src/test/scala/org/broadinstitute/hail/methods/ExprSuite.scala

```diff
@@ -479,7 +479,7 @@ class ExprSuite extends SparkSuite {
       eval[IndexedSeq[Int]]("""[1] + [2,3,4] """)
     }
 
-    interceptFatal("invalid arguments") {
+    interceptFatal("No function found") {
```

Comment:
I am disappointed that the new error messages are less detailed. Previously, we would suggest types that are valid for this name. Now, we just write out the name and the argument types that the user provided. As a further improvement to the registry, we could add a map from function name to valid argument types and list these in the error message.